manifest/com.nalathethird/bHapticsManagerFix/info.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "bHapticsManagerFix",
+  "id": "com.nalathethird.bHapticsManager",
+  "description": "A mod to fix issues with the bHaptics Manager.",
+  "category": "Bug Workaround",
+  "sourceLocation": "https://github.com/nalathethird/Resonite-bHapticsManagerFix",
+  "versions": {
+    "1.0.0": {
+      "releaseUrl": "https://github.com/nalathethird/Resonite-bHapticsManagerFix/releases/tag/1.0.0",
+      "dependencies": {
+        "com.nalathethird.bHapticsLib": {
+          "version": "^1.0.8"
+        }
+      },
+      "artifacts": [
+        {
+          "url": "https://github.com/nalathethird/Resonite-bHapticsManagerFix/releases/download/1.0.0/bHapticsManager.dll",
```

Comment:
Link in mod is a 404 from renaming the repo a few times. `https://github.com/nalathethird/Resonite-bHapticsManagerFix` should be `https://github.com/nalathethird/Resonite-bHapticsManagerFix`